### PR TITLE
Update uid2 doc after euid work

### PR DESCRIPTION
**Pull request by caroline-ttd**

### CODE REVIEW

File: docs/guides/integration-mobile-client-server.md [hunk 1]
```diff
@@ -48,10 +48,10 @@ You'll need to complete the following steps:
 
 ## Mobile SDK Version
 
-This guide provides instructions for using version 1.2.0 or higher of either of these UID2 mobile SDKs:
+This guide provides instructions for using either of these UID2 mobile SDKs:
 
-- SDK for Android
-- SDK for iOS
+- SDK for Android(version 1.5.0 or later)
```
Comment (genwhittTTD) [inline docs/guides/integration-mobile-client-server.md:53]: Space before the open quotes here:
- SDK for Android (version 1.5.0 or later)
- SDK for iOS (version 1.7.0 or later)

File: docs/guides/integration-mobile-client-side.md [hunk 2]
```diff
@@ -49,10 +49,10 @@ To integrate with UID2 client-side, you'll need to complete the following steps:
 
 ## Mobile SDK Version
 
-This guide provides instructions for using version 1.2.0 or higher of either of these UID2 mobile SDKs:
+This guide provides instructions for using either of these UID2 mobile SDKs:
 
-- SDK for Android
-- SDK for iOS
+- SDK for Android(version 1.5.0 or later)
+- SDK for iOS(version 1.7.0 or later)
```
Comment (genwhittTTD) [inline docs/guides/integration-mobile-client-side.md:55]: Same here, just add a space:
- SDK for Android (version 1.5.0 or later)
- SDK for iOS (version 1.7.0 or later)

File: docs/guides/integration-mobile-client-side.md [hunk 3]
```diff
@@ -67,7 +67,8 @@ Follow the applicable instructions, for Android or iOS:
 
 1. Check out the main branch of the [SDK for Android source code repository on GitHub](https://github.com/IABTechLab/uid2-android-sdk/tree/main).
 1. In Android Studio (check the version required in the [Minimum Requirements](../sdks/sdk-ref-android.md#minimum-requirements) section in the SDK for Android Reference Guide), open the directory that you checked out.
-1. Run the **dev-app** app.
+1. set `uid2_environment_euid` to `false` in [AndroidManifest.xml](https://github.com/IABTechLab/uid2-android-sdk/blob/main/dev-app/src/main/AndroidManifest.xml)
```
Comment (genwhittTTD) [inline docs/guides/integration-mobile-client-side.md:70]: Add a period (.) at the end of the sentence.

File: docs/guides/integration-mobile-client-side.md [hunk 4]
```diff
@@ -80,6 +81,14 @@ Follow the applicable instructions, for Android or iOS:
    ```js
    Development/UID2SDKDevelopmentApp/UID2SDKDevelopmentApp.xcodeproj
    ```
+1. Set the `UID2EnvironmentEUID` key to `NO` in `Development/UID2SDKDevelopmentApp/UID2SDKDevelopmentApp/Info.plist` in Xcode's editor. Alternatively you can use `plutil` from the command line:
+   ```console
+   plutil -replace UID2EnvironmentEUID -bool NO Development/UID2SDKDevelopmentApp/UID2SDKDevelopmentApp/Info.plist
+   ```
+   to use a EUID environment,
```
Comment (genwhittTTD) [inline docs/guides/integration-mobile-client-side.md:88]: To use a EUID environment:

(initial cap, different ending punctuation)

**Review by genwhittTTD — COMMENTED** [review]
Thanks! Few small language updates.

**Comment by caroline-ttd** [comment]
> Thanks! Few small language updates.

Thanks a lot Gen

**Review by sunnywu — APPROVED** [review]

**Pull request merged**